frontend/ui/elements/menu_keyboard_layout.lua

```diff
@@ -1,17 +1,67 @@
 local CheckButton = require("ui/widget/checkbutton")
+local CheckMark = require("ui/widget/checkmark")
+local Device = require("device")
 local FFIUtil = require("ffi/util")
+local Font = require("ui/font")
 local Language = require("ui/language")
+local Size = require("ui/size")
+local TextWidget = require("ui/widget/textwidget")
 local UIManager = require("ui/uimanager")
 local VirtualKeyboard = require("ui/widget/virtualkeyboard")
+local Screen = Device.screen
+local T = require("ffi/util").template
 local dbg = require("dbg")
+local logger = require("logger")
 local util = require("util")
 local _ = require("gettext")
 
 local input_dialog, check_button_bold, check_button_border, check_button_compact
 
+local function getActivatedKeyboards(compact)
```

Comment:
`compact` is not used now. Keep it for future?

Reply:
Yes, I would like to keep it. Maybe I should add a comment on that? What do you think?

Reply:
Yes, agree.